### PR TITLE
doc: updated more plugins link

### PR DESCRIPTION
**Pull request by saranyaeaswaran**
## **Summary**
The link provided under https://github.com/docsifyjs/docsify/blob/develop/docs/plugins.md#more-plugins is not working, updated this with working link

## **What kind of change does this PR introduce?**
Docs

## **For any code change,**

- [ ] Related documentation has been updated if needed
- [ ] Related tests have been updated or tests have been added

## **Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

## **Tested in the following browsers:**

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

### CODE REVIEW

File: docs/plugins.md [hunk 1]
```diff
@@ -203,4 +203,4 @@ Provided by [@jhildenbiddle](https://github.com/jhildenbiddle/docsify-tabs).
 
 ## More plugins
 
-See [awesome-docsify](awesome?id=plugins)
+See [awesome-docsify](https://github.com/docsifyjs/awesome-docsify/blob/master/README.md#plugins)
```
Comment (sy-records) [inline docs/plugins.md:206]: Thank you for your contribution.

But this modification seems to be incorrect. 

Can't you access this link? https://docsify.js.org/#/awesome?id=plugins

Comment (saranyaeaswaran) [reply docs/plugins.md:206]: Hi @sy-records Yes I can access this link [docsify.js.org/#/awesome?id=plugins](https://docsify.js.org/#/awesome?id=plugins), should I update this link as the correct one in this change? Thanks for the correction.

Comment (sy-records) [reply docs/plugins.md:206]: No, you don't have to revise.
If you can access it, it means the original link is working and I will close your PR.
Thanks again.

**Comment by sy-records** [comment]
You should use our documentation access for the best experience, rather than viewing Markdown source files in GitHub.
https://docsify.js.org/#/plugins?id=more-plugins

**Comment by trusktr** [comment]
Thanks Saranya, yeah, some things only work when you visit the website. The stuff on GitHub is the "source code".

**Pull request closed without merging**